### PR TITLE
feat(supportability): add success log to the dump commands

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Signed-off-by: Vandana Varakantham <vandana.varakantham@datacore.com>

### CODE REVIEW

File: k8s/supportability/src/collect/resource_dump.rs [hunk 1]
```diff
@@ -217,6 +217,7 @@ impl ResourceDumper {
             errors.push(Error::EtcdDumpError(e));
         });
         log("Completed collection of mayastor specific resources from Etcd service".to_string());
+        log("Completed collection of dump information".to_string());
```
Comment (avishnu) [inline k8s/supportability/src/collect/resource_dump.rs]: Not needed as we already have a completion log above.

**Comment by datacore-vvarakantham** [comment]
bors try

**Comment by datacore-vvarakantham** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors d+

**Comment by tiagolobocastro** [comment]
@datacore-vvarakantham you should be able to "bors try" for this PR.
@avishnu probably need to put @datacore-vvarakantham on the maintainers group to get bors permissions by default?

**Comment by datacore-vvarakantham** [comment]
bors try

**Review by Abhinandan-Purkait — APPROVED** [review]

**Pull request closed without merging**